src/Elastic.Apm.Azure.Functions/Elastic.Apm.Azure.Functions.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <Nullable>enable</Nullable>
+
+    <AssemblyName>Elastic.Apm.Azure.Functions</AssemblyName>
+    <RootNamespace>Elastic.Apm.Azure.Functions</RootNamespace>
+    <PackageId>Elastic.Apm.Azure.Functions</PackageId>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Elastic.Apm\Elastic.Apm.csproj"/>

```

Comment:
I think we should take dependency on more packages and turn on other listeners - the other Azure packages (CosmosDb, ServiceBus, Storage) are especially interesting here, since those are more likely to be used in an Azure function.

Reply:
Good point. However, let's do this in another iteration as you suggested 👍 .